src/validation.cpp

```diff
@@ -288,6 +288,8 @@ static unsigned int GetBlockScriptFlags(const CBlockIndex* pindex, const Consens
 static void LimitMempoolSize(CTxMemPool& pool, CCoinsViewCache& coins_cache, size_t limit, std::chrono::seconds age)
     EXCLUSIVE_LOCKS_REQUIRED(pool.cs, ::cs_main)
 {
+    AssertLockHeld(pool.cs);
+    AssertLockHeld(::cs_main);
```

Comment:
nit: Reorder here and in `EXCLUSIVE_LOCKS_REQUIRED`?

Reply:
Done (thanks!)